examples/connext_dds/group_coherent_presentation/c++11/group_coherent.idl

```diff
@@ -0,0 +1,20 @@
+struct CheckpointPlace {
```

Comment:
Most of our examples use pretty bad names for IDL files and type names. They make the code confusing. I've started this approach: name the IDL file <feature>Example.idl, and define a module also called `module <feature>Example`. Then use meaningful names for the types in that module (you're alrerady doing that in this case) or just call the struct `HelloWorld` if the type is irrelevant (not this case).

So I would call it `GroupCoherentExample.idl` and then define:

```
module GroupCoherentExample
{
    struct CheckPointPlace { ... };
    ...
};
```